tokio-codec/src/framed_read.rs

```diff
@@ -16,7 +16,7 @@ pub struct FramedRead<T, D> {
     inner: FramedRead2<Fuse<T, D>>,
 }
 
-pub struct FramedRead2<T> {
+pub(crate) struct FramedRead2<T> {
```

Comment:
I naively want to say this was a relic to not introduce a breaking change? Maybe we should just make this FramedRead? cc @carllerche